Fika.Core/Coop/Utils/FikaGroupUtils.cs

```diff
@@ -0,0 +1,28 @@
+namespace Fika.Core.Coop.Utils
+{
+    public static class FikaGroupUtils
+    {
+        public static MatchmakerPlayerControllerClass GroupController { get; set; }
+        public static bool InGroup => GroupSize > 1;
+        public static int GroupSize => GroupController?.GroupPlayers?.Count ?? 1;
+        public static string GroupId => GroupController?.Group?.Owner?.Value?.Id ?? FikaBackendUtils.Profile.ProfileId;
+        
+        public static bool IsGroupLeader
+        {
+            get
+            {
+                if (GroupController?.Group == null) return true;
```

Comment:
```suggestion
                if (GroupController?.Group == null)
                {
                    return true;
                }
```
Per code guidelines, keep the scope consistent. Same with the rest in this `get`